src/client/styles/brand.css

```
@@ -0,0 +1,105 @@
+/* SortedMate SEO brand layer.
+ *
+ * Loaded after app.css (see src/routes/__root.tsx) and deliberately kept out
+ * of it: upstream owns app.css, this file is ours, so upstream merges never
+ * conflict here. It overrides the daisyUI theme variables that the two
+ * upstream themes define ("openseo" / "openseo-dark" stay as internal ids —
+ * nobody sees them) with the SortedMate tokens from the main app's index.css.
+ *
+ * Industrial-Modern | Black & Yellow | Bold & Confident.
+ */
+
+@import url("https://fonts.googleapis.com/css2?family=DM+Sans:ital,opsz,wght@0,9..40,400;0,9..40,500;0,9..40,600;0,9..40,700;1,9..40,400&family=JetBrains+Mono:wght@400;500;600&display=swap");
```

Stylelint (17.14.0)

[error] Expected "url("https://fonts.googleapis.com/css2?family=DM+Sans:ital,opsz,wght@0,9..40,400;0,9..40,500;0,9..40,600;0,9..40,700;1,9..40,400&family=JetBrains+Mono:wght@400;500;600&display=swap")" to be ""https://fonts.googleapis.com/css2?family=DM+Sans:ital,opsz,wght@0,9..40,400;0,9..40,500;0,9..40,600;0,9..40,700;1,9..40,400&family=JetBrains+Mono:wght@400;500;600&display=swap"" (import-notation)

(import-notation)